.github/workflows/minimal.yml

```diff
@@ -1,172 +0,0 @@
-name: Minimal-CI
```

Comment:
Why did you delete this file ? :)

Reply:
I don't think we need the minimal CI. All jobs run in the main CI/Epidata CI.

Reply:
The minimal CI is more for the developer to test branches that are WIP. Also, it runs faster and only tests basic functionality.
So we shouldn't delete them.